### ISSUE TITLE
Add analytics

### ISSUE THREAD

**Issue by dhowe** [dhowe/Website#238]
are there decent analytics we can use other than google?

**Comment by cqx931** [comment]
Two open source alternatives that can be installed locally. 
openwebanalytics: http://www.openwebanalytics.com/
countly: https://github.com/Countly/countly-server

**Comment by cqx931** [comment]
Countly installation script assumes it is running on a fresh, decent Ubuntu/CentOS/RHEL Linux without any services listening on port 80 or 443 (which should also be open to incoming traffic), and takes care of every library and software required to be installed for Countly to run.

**Comment by dhowe** [comment]
Can you add this code to the relevant pages on the site (ignore wordpress for now):

The Site ID for https://rednoise.org is 4b0abd7f174679a5dc058eb6ebdd336d

To track page views using Javascript, cut and paste this tracking tag into the HTML of your web pages. Learn more about how to use OWA's [Javascript tracking API](http://wiki.openwebanalytics.com/index.php?title=Javascript_Invocation) to track your web site and pages.


```
<!-- Start Open Web Analytics Tracker -->
<script type="text/javascript">
//<![CDATA[
var owa_baseUrl = 'https://rednoise.org/owa/';
var owa_cmds = owa_cmds || [];
owa_cmds.push(['setSiteId', '4b0abd7f174679a5dc058eb6ebdd336d']);
owa_cmds.push(['trackPageView']);
owa_cmds.push(['trackClicks']);

(function() {
    var _owa = document.createElement('script'); _owa.type = 'text/javascript'; _owa.async = true;
    owa_baseUrl = ('https:' == document.location.protocol ? window.owa_baseSecUrl || owa_baseUrl.replace(/http:/, 'https:') : owa_baseUrl );
    _owa.src = owa_baseUrl + 'modules/base/js/owa.tracker-combined-min.js';
    var _owa_s = document.getElementsByTagName('script')[0]; _owa_s.parentNode.insertBefore(_owa, _owa_s);
}());
//]]>
</script>
```

**Comment by dhowe** [comment]
implemented -- lets see how it is working...

**Comment by cqx931** [comment]
Yes, I can verify the tracking code on the new detail pages for the published site. By events do you mean the wordpress site?

**Comment by dhowe** [comment]
Yes

**Comment by cqx931** [comment]
Just added it to the wordpress home page, can you check whether it works for OWA at all? Potentially it can also be added to all wordpress pages if needed.

**Comment by dhowe** [comment]
I don't see it, after also visiting it myself. I think something more may be required to get it working with wordpress. Did you read [this](https://www.anetcomputers.com/open-web-analytics-how-to-install-owa-wordpress-plugin/)?

Also, we need to figure out how to set the 'page-type' for each of the pages -- it is currently 'unset' everywhere.

Ping me when working again and I can give you access to owa

**Comment by cqx931** [comment]
page-type: https://github.com/Open-Web-Analytics/Open-Web-Analytics/wiki/Filters

**Comment by cqx931** [comment]
` owa_cmds.push(['setPageType', 'Home']);` doesn't seem to work as expected, related issues: https://github.com/Open-Web-Analytics/Open-Web-Analytics/issues/366

**Comment by dhowe** [comment]
perhaps it is bc our owa is not updated (see the messages on lower right in the admin ui)
if you feel comfortable, try the updates...

**Comment by dhowe** [comment]
did you try: window.OWATracker.setPageType('your-page-type') ?

**Comment by cqx931** [comment]
wpr works fine now, get stucked on setPageType.
I have created a ticket for this in their github repo, will see if anyone answers there.

**Comment by cqx931** [comment]
I think this is what we need for setPageType to work:

> Go into the DB and alter it manually in the owa_document table. I think if you've visited the site before you added the owa command it won't update the site info and remain (not set) or whatever it is.
